src/models/text.ts

```diff
@@ -187,12 +193,51 @@ class SignInterpretation {
         }
     }
 
+    public findAttributeIndex(attributeValueId: number) {
+        return this.attributes.findIndex(attr => attr.attributeValueId === attributeValueId);
+    }
+
     public get isReconstructed(): boolean {
         return !!this.attributes.find(attr => attr.attributeString === 'is_reconstructed');
```

Comment:
I know this is old code, and !! is cute, but I tend to prefer "!== undefined" to Javascript coercion.  A guy can dream that someday they will clean up all the coercion/equality junk in Javascript ... ok, I know it won't really happen.